File: FileCreate/Python/PythonFile.py
import sys

code = """#!/bin/env python


def main():
    print("")








if __name__ == "__main__":
    main()

"""

def main():
    arg1 = sys.argv[1]
    print(arg1)

    if arg1.endswith(".py"):
        f = open(arg1, "w")
        f.write(code)
        f.close()
    else:
        print("Python file should have .py extension!")


if __name__ == "__main__":
    main()

File: FileCreate/Golang/PythonFile.go
package main

import (
	"fmt"
	"log"
	"os"
	"strings"
)

func main() {
	FileCreate()
}

func FileCreate() {
	code := `#!/bin/env python

def main():
	print("")

if __name__ == "__main__":
	main()
`
	fileName := os.Args[1]
	isextention := strings.HasSuffix(fileName, ".py")
	if isextention {
		file, err := os.Create(fileName)

		if err != nil {
			log.Fatal(err)
		}

		defer file.Close()

		_, err2 := file.WriteString(code)

		if err2 != nil {
			log.Fatal(err2)
		}

		fmt.Println("Done!")

	} else {
		fmt.Println("Python File should have extension of .py")
	}

}
